pkg/common/logging/api/loggerapi.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+// Level defines all available log levels for logging messages.
+type Level int
+
+// Log levels.
+const (
+	CRITICAL Level = iota
+	ERROR
+	WARNING
+	INFO //default logging level
+	DEBUG
+)
+
+//Logger - Standard logger interface
+type Logger interface {
+
+	//Fatalf is critical fatal logging, should possibly followed by system shutdown
```

Comment:
"should possibly followed by system shutdown" is bit confusing. May be we can use standard Go library doc message (to be consistent with the standard lib).

"Fatalf is equivalent to Printf() followed by a call to os.Exit(1)" - https://godoc.org/log#Fatalf

Reply:
Fixed